### ISSUE TITLE
Hyperparameter-tuning

### ISSUE THREAD

**Issue by tatiana-boura** [eneribnk/Tasks_plasmAI#5]
Tune model's hyperparameters (learning rate, batch size, weight decay, optimizer etc.)  as well as its architecture (layers, activation functions etc.) with respect to minimal validation loss.

The loss should be chosen carefully, based on some naive testing - preferably on the validation set. For example, weighted MSE may be better than simple MSE.

The tuning should be done manually or using a library. 

For educational pursposes manual tuning should be implemented and compared to the automated one for the values that can be tuned using both methods.
